frontend/src/components/interests/category/category.module.css

```diff
@@ -0,0 +1,7 @@
+.tagsContainer {
```

Comment:
can we rename the file to `.scss` instead of `.css`

Reply:
think they also `@use '~/src/styles/theme';` on all css files

Reply:
Yup sure, made the required changes!